ex02/src/my_strcmp.c

```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <assert.h>
+#include <string.h>
+#include "test.h"
+
+int my_strcmp(char *str1, char *str2)
+{
+  while (*str1 == *str2)
+{
+  if (*str1 == '\0' || *str2 == '\0')
+  break;
+  str1++;
+  str2++;
+}
+  return *str1 - *str2 > 0 ? 1: *str1 - *str2 < 0 ? -1: 0; 
```

Comment:
I suggest you to simplify this line of code because It is not readable
Use ternary operator if you have 2 conditions